dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESMappingAPITest.java

```diff
@@ -104,9 +105,9 @@ public void testLoadRelationshipFields_whenUsingLegacyRelationships_shouldSucces
 
             assertNotNull(esMap);
             assertEquals(commentsContentlet.getIdentifier(),
-                    esMap.get("News-Comments").toString().trim());
+                    ((List)esMap.get("News-Comments")).get(0));
```

Comment:
You can use List.class.cast(esMap.get("News-Comments")).get(0)
I think it s a bit more readable